charts/fleet/README.md

```diff
@@ -31,7 +31,7 @@ apiVersion: v1
 metadata:
   name: fleet
   namespace: fleet
-stringData:
+data:
```

Comment:
`stringData` is actually more appropriate here because the data is not base64 encoded in the example.  See https://kubernetes.io/docs/concepts/configuration/secret/#restriction-names-data for details.

Reply:
I understand. I converted the certificate to base64 when using it, so using "data" will result in an error.